homeassistant/components/nws/weather.py

```diff
@@ -193,16 +182,7 @@ def wind_speed(self):
         wind_km_hr = None
         if self.observation:
             wind_km_hr = self.observation.get("windSpeed")
-        if wind_km_hr is None:
-            return None
```

Comment:
For consistency in docstrings if you are going to change them, put the unit in the docstring.